components/browse-decks/DeckSearch.js

```diff
@@ -8,13 +9,22 @@ const DeckSearch = styled.div`
   justify-content: flex-start;
   align-items: flex-start;
   margin: 10px;
+  flex: 0 0 30%;
+
+  @media screen and (max-width: ${screenSizes.smallPhone.max}) {
+    justify-content: center;
+    align-items: center;
+  }
 `
 const SearchBar = styled.div`
   display: flex;
   flex-direction: row;
   padding-bottom: 10px;
   height: 50px;
-  width: 100%;
+  width: 280px;
```

Comment:
Why is it like this? Is this because that the pills have this size?

Reply:
yes, width 100% didn't want to work